Xamarin.Forms.Platform.UAP/ImageRenderer.cs

```diff
@@ -189,5 +195,39 @@ void UpdateAspect()
 				Element.SetIsLoading(false);
 			}
 		}
+
+		void UpdateImageSearchDirectory()
+		{
+			if (_disposed || Element == null || Control == null)
+			{
+				return;
+			}
+
+			var imageSearchDirectory = Application.Current.OnThisPlatform().GetImageSearchDirectory();
+
+			if(!string.IsNullOrEmpty(imageSearchDirectory))
+			{
+				var source = Element.Source;
+
+				if (source is FileImageSource)
```

Comment:
Can we use some C# features here:  if (source is FileImageSource fileSource && fileSource != null)

Reply:
Sure,. changed.